spec/Index.spec.js

```diff
@@ -0,0 +1,14 @@
+var path = require('path');
+
+describe('Directive Test', function() {
+	browser.get('http://localhost:8080');
+
+	it('should display the info correctly', function() {
+		var directive = element.all(by.css('contact-card'));
```

Comment:
update this variable name to be `component` - it will make the tests a little bit more clear as to what functionality we're expecting